src/components/Modals/Onboarding/OnboardingReview.vue

```diff
@@ -116,6 +119,8 @@
                 }}</label>
               </div>
             </div>
+            <!-- 
+              TODO: BRING THIS BACK POSTLAUNCH
             <div class="alignCenter">
               <label class="onboarding-label">Credit</label>
               <div v-for="(exam, index) in onboardingData.exam" :key="'APCredit' + index">
```

Comment:
Discussed with @einc and we agree with @SamChou19815 that deleting the code instead of commenting it out would prob be better & cleaner. Sorry about that! Can you delete the commented out code in your PR?